test/lib/ufe/CMakeLists.txt

```diff
@@ -40,18 +40,12 @@ endif()
 # copy ufe tests to ${CMAKE_CURRENT_BINARY_DIR} and run them from there
 add_custom_target(${TARGET_NAME} ALL)
 
-mayaUsd_copyDirectory(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                             DIRECTORY test-samples)
-mayaUsd_copyDirectory(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                             DIRECTORY ufeScripts)
-mayaUsd_copyDirectory(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                             DIRECTORY ufeTestUtils)
-mayaUsd_copyDirectory(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                             DIRECTORY ufeTestPlugins)
-mayaUsd_copyFiles(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                         FILES ${test_script_files})
-mayaUsd_copyFiles(${TARGET_NAME} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
-                         FILES ${test_support_files})
```

Comment:
It looks like using the new form of `mayaUsd_copyDirectory()` is actually going to result in *slightly* different behavior here. I was initially going to suggest that I don't think we need to define `test_support_files` anymore, but I then realized that we're now going to **always** install everything in this directory, even if the Maya release you're building for does not support V2 or a compatible preview version of UFE. Any concerns about that?

Reply:
test_support_files can be removed if not used. This can be cleaned up in another PR.  

We unconditionally always want to install all of python tests and resources from source dir to build directory regardless of Maya or UFE version being used.  We could exclude things from being copied during mayaUsd_copyDirectory but that's redundant and doesn't gain us anything at the end.

@mattyjams does this answer your question?

Reply:
Yup, thanks. No real concerns from me. I just wanted to raise the point that this isn't exactly equivalent to what was here before. But as you said, what files get installed doesn't really matter as long as we're still only registering the right set of tests, which is definitely still the case. Thanks!